### ISSUE TITLE
Braking change introduced by minor version 0.3.13

### ISSUE THREAD

**Issue by Lesiuk** [alexcrichton/socket2-rs#93]
https://docs.rs/socket2/0.3.12/socket2/struct.Domain.html
https://docs.rs/socket2/0.3.13/socket2/struct.Domain.html

functions missing
`socket2::Domain::ipv4()`
`socket2::Domain::ipv6()`

**Comment by rossmacarthur** [comment]
There are also breaking changes on 

https://docs.rs/socket2/0.3.12/socket2/struct.Protocol.html
https://docs.rs/socket2/0.3.12/socket2/struct.Type.html

**Comment by jkoudys** [comment]
Getting all these building actix:
```
error[E0599]: no function or associated item named `ipv4` found for struct `socket2::Domain` in the current scope
   --> /Users/joshuakoudys/.cargo/registry/src/github.com-1ecc6299db9ec823/actix-server-1.0.3/src/builder.rs:491:43
    |
491 |         net::SocketAddr::V4(_) => Domain::ipv4(),
    |                                           ^^^^ function or associated item not found in `socket2::Domain`

error[E0599]: no function or associated item named `ipv6` found for struct `socket2::Domain` in the current scope
   --> /Users/joshuakoudys/.cargo/registry/src/github.com-1ecc6299db9ec823/actix-server-1.0.3/src/builder.rs:492:43
    |
492 |         net::SocketAddr::V6(_) => Domain::ipv6(),
    |                                           ^^^^ function or associated item not found in `socket2::Domain`

error[E0599]: no function or associated item named `stream` found for struct `socket2::Type` in the current scope
   --> /Users/joshuakoudys/.cargo/registry/src/github.com-1ecc6299db9ec823/actix-server-1.0.3/src/builder.rs:494:44
    |
494 |     let socket = Socket::new(domain, Type::stream(), Some(Protocol::tcp()))?;
    |                                            ^^^^^^ function or associated item not found in `socket2::Type`

error[E0599]: no function or associated item named `tcp` found for struct `socket2::Protocol` in the current scope
   --> /Users/joshuakoudys/.cargo/registry/src/github.com-1ecc6299db9ec823/actix-server-1.0.3/src/builder.rs:494:69
    |
494 |     let socket = Socket::new(domain, Type::stream(), Some(Protocol::tcp()))?;
    |                                                                     ^^^ function or associated item not found in `socket2::Protocol`

error: aborting due to 4 previous errors
```

**Comment by rossmacarthur** [comment]
@stjepang there are still breaking changes on 

docs.rs/socket2/0.3.12/socket2/struct.Protocol.html
docs.rs/socket2/0.3.12/socket2/struct.Type.html

that were not fixed in #96
